Specification-Draft.MD

```diff
@@ -20,7 +20,7 @@ FDC3 Intents define a standard set of nouns and verbs that can be used to put to
 * Atomic
     * There should be clear lines in a workflow where the object begins and ends
 * Stateless
-    * Workflows should not require callbacks or endpoints to maintain references to each other.  Once an intent is passed to an endpoint - it controls the rest of that workflow. 
+    * Workflows should not require callbacks or endpoints to maintain references to each other.  With the exception of specifically data-driven intents, once an intent is passed to an endpoint - it controls the rest of that workflow.
```

Comment:
Just saying this isn't enough, in my opinion. The doc needs to clearly set out the difference between the two.